ods_ci/tests/Tests/0600__distributed_workloads/0601__workloads_orchestration/test-smoke.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Documentation     Smoke tests for Workloads Orchestration
+Library           Process
+Resource          ../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
+
+
+*** Test Cases ***
+Kueue smoke test
+    [Documentation]    Check that Kueue deployment and its service are up and running
+    [Tags]    Smoke
+    ...       DistributedWorkloads
```

Comment:
`WorkloadsOrchestration` tag is missed here

Reply:
Fixed.